wai-extra/Network/Wai/Middleware/HttpAuth.hs

```diff
@@ -40,7 +40,15 @@ type CheckCreds = ByteString
 basicAuth :: CheckCreds
           -> AuthSettings
           -> Middleware
-basicAuth checkCreds AuthSettings {..} app req sendResponse = do
+basicAuth checkCreds = basicAuth' (\_ -> checkCreds)
+
+-- | Like 'basicAuth', but also passes a request to the authentication function.
+--
+-- Since 3.0.19
```

Comment:
Can you use `@since` syntax instead? I know that the library still uses `Since` in many places, but I'm trying to update it bit by bit.